libs/designsystem/reorder-list/src/reorder-list.component.scss

```
@@ -61,6 +61,14 @@ ion-backdrop {
   border-bottom: 20px transparent solid;
 }
 
+ion-reorder-group > div {
+  border-bottom: 20px transparent solid;
+  
+  &:last-child {
+    border-bottom: transparent solid;
```

Consider only changing what needs to change :) 
```suggestion
    border-bottom-width: 0;
```